src/service_discovery/mod.rs

```diff
@@ -287,7 +289,7 @@ mod tests {
             unwrap!(
                 el1.send(CoreMessage::new(move |core, poll| {
                     unwrap!(
-                        ServiceDiscovery::start(core, poll, listeners_1, token_1, 65_530),
+                        ServiceDiscovery::start(core, poll, listeners_1, token_1, 0xFFFA),
```

Comment:
@m-cat I think we should disable this clippy lint after all.
E.g. this particular case is the port number and it makes not much sense to have it as a hexadecimal. So I'd agree with @canndrew and @ustulation about this lint not being very helpful at least for Crust.